src/cmake-tools.ts

```diff
@@ -657,7 +730,11 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
    * Implementation of `cmake.quickStart`
    */
   public async quickStart(): Promise<Number> {
-    const drv = await this._cmakeDriver;
+    const drv = await this.getCMakeDriverInstance();
+    if (!drv) {
+      return -2;
```

Comment:
I think this should be returned as a `Promise`:
```ts
return Promise.resolve(-2);
```

Reply:
Style guide question